helm/mojaloop-barebone/values.yaml

```diff
@@ -0,0 +1,129 @@
+ph-ee-engine:
+  zeebe-cluster-helm:
+    global:
+      elasticsearch:
+        host: "ph-ee-elasticsearch"
+    image:
+      repository: camunda/zeebe
+      tag: 1.1.0
+
+    clusterSize: "1"
+    partitionCount: "1"
+    replicationFactor: "1"
+    
+    elasticsearch:
+      enabled: false
+    kibana:
+      enabled: false
+
+    extraInitContainers: |
+      - name: init-ph-ee-kafka-exporter
+        image: busybox:1.28
+        command: ['/bin/sh', '-c']
+        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.0.0-SNAPSHOT.jar"; ls -al /exporters/']
+        volumeMounts:
+        - name: exporters
+          mountPath: /exporters/
+
+  zeebe-operate-helm:
+    enabled: true
```

Comment:
this was disabled in the template, is this needed?

Reply:
This change is not part of this PR.